base/nvti.c

```diff
@@ -763,6 +763,44 @@ nvti_tag (const nvti_t *n)
   return (n ? n->tag : NULL);
 }
 
+/**
+ * @brief Get a tag value by a tag name.
+ *
+ * @param n The NVT Info structure from where to search for the tag name.
+ *
+ * @param name The name of the tag for which to return the value.
+ *
+ * @return The tag value string as a copy or NULL if not found.
+ *         Needs to be free'd.
+ */
+gchar *
+nvti_get_tag (const nvti_t *n, const gchar *name)
+{
+  gchar **split, **point;
+
+  if (n->tag == NULL)
```

Comment:
I would check for a null pointer here, before accessing n.
```suggestion
  if (n && n->tag == NULL)
```